index.js

```diff
@@ -131,6 +135,8 @@ module.exports = {
   // wrapped in a shim that stops it from exporting a global and instead turns it into a module
   // that can be used by the Ember app.
   treeForBrowserFetch() {
+    const needsFetchPolyfill = !this._checkSupports('fetch');
```

Comment:
The issue here is we don't just polyfill `fetch`, we polyfill `AbortController` as well. And they have different browser support. `caniuse also has a [note](https://caniuse.com/#search=fetch)

> Safari has window.AbortController defined in the DOM but it's just a stub, it does not abort requests at all. The same issue also affects Chrome on IOS and Firefox on IOS because they use the same WebKit rendering engine as Safari.

which is detected by our polyfill lib [programmatically](https://github.com/mo/abortcontroller-polyfill/commit/256fde6b551678bbb72a7821c1fdbb734bfbe84a). I'm only brain debugging this... so apparent we need to verify API response or we come up with a better way.

But I totally agree if `preferNative && supportNativeFetchAndAbortController && !alwaysIncludePolyfill` we should just strip off the polyfill.

Or if apart from `preferNative` and `alwaysIncludePolyfill` we can provide 2 separate group of these for `fetch` and `abortController` with reasonable defaults. User can have finer-grained control of this addon. What do you think?

Reply:
True, I did not think about this - Can I Use has just recently updated this to correctly state that abortcontroller does not work in Safari. When I looked the last time, it seemed as if all fetch-supporting browsers also supported abortcontroller - which is clearly not the case.

We can just load the necessary polyfills - e.g. fetch & abortcontroller, or only abortcontroller, or none!